crates/xilem_masonry/src/view/button.rs

```diff
@@ -53,4 +44,22 @@ where
             ChangeFlags::UNCHANGED
         }
     }
+
+    fn message(
+        &self,
+        _view_state: &mut Self::ViewState,
+        id_path: &[ViewId],
+        message: Box<dyn std::any::Any>,
+        app_state: &mut State,
+    ) -> crate::MessageResult<Action> {
+        debug_assert!(
```

Comment:
I've been doing a bit of thinking here, and I'm wondering if action views *should* get an entry in the id path, entirely for logging purposes (i.e. when printing out an idpath, we can see what kind of view it is targetting)

But that's out-of-scope for this PR.

Reply:
Good point, maybe implement that behavior directly in `ViewCx`? I mean generally some debugging helpers.